### PR TITLE
Improve docs for "isotropic" arg in RandomAffine

### LINKED ISSUE #1158: RandomAffine raises an error when isotropic=True and 3 elements are given for scales
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug summary

In transforms.augmentation.spatial.random_affine, if the isotropic argument is given as True and 3 values are given as scales, _parse_scales_isotropic function will raise an error

### Code for reproduction

```python
spatial_transforms = {tio.RandomAffine(scales=(0.3, 0.3, 0.3), label_keys=['soma', 'trace'], isotropic=True, degrees=0, center='image', image_interpolation='bspline', label_interpolation='label_gaussian'): 0.4}
```

### Actual outcome

```console
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "./venv/lib/python3.11/site-packages/torchio/transforms/augmentation/spatial/random_affine.py", line 127, in __init__
    _parse_scales_isotropic(scales, isotropic)
  File ./venv/lib/python3.11/site-packages/torchio/transforms/augmentation/spatial/random_affine.py", line 447, in _parse_scales_isotropic
    raise ValueError(message)
ValueError: If "isotropic" is True, the value for "scales" must have length 1 or 2, but "(0.3, 0.3, 0.3)" was passed
```

### Error messages

```python-traceback
If "isotropic" is True, the value for "scales" must have length 1 or 2, but "3" was passed
```

### Expected outcome

No error should be raised as in the isotropic mode the only condition should be met is s1==s2==s3 :)

### System info

```shell
Platform:   Linux-6.1.0-17-amd64-x86_64-with-glibc2.36
TorchIO:    0.19.6
PyTorch:    2.2.1+cu121
SimpleITK:  2.3.1 (ITK 5.3)
NumPy:      1.26.4
Python:     3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0]
```

### PR DESCRIPTION
**Pull request by AminAlam**
Fixes #1158.

**Description**

**Checklist**

- [x] I have read the [`CONTRIBUTING`](https://github.com/fepegar/torchio/blob/main/CONTRIBUTING.rst) docs and have a developer setup (especially important are `pre-commit`and `pytest`)
- [x] Non-breaking change (would not break existing functionality)
- [ ] Breaking change (would cause existing functionality to change)
- [ ] Tests added or modified to cover the changes
- [x] Integration tests passed locally by running `pytest`
- [x] In-line docstrings updated
- [x] Documentation updated, tested running `make html` inside the `docs/` folder
- [x] This pull request is ready to be reviewed

### CODE REVIEW

File: src/torchio/transforms/augmentation/spatial/random_affine.py [hunk 1]
```diff
@@ -442,7 +446,9 @@ def _parse_scales_isotropic(scales, isotropic):
     if isotropic and len(scales) in (3, 6):
         message = (
             'If "isotropic" is True, the value for "scales" must have'
-            f' length 1 or 2, but "{scales}" was passed'
+            f' length 1 or 2, but "{scales}" was passed.'
+            ' If you want to set isotropic scaling, use a single value or two values as a range'
+            ' for scaling factor. Refer to the documentation for more information.'
```
Comment (fepegar) [inline src/torchio/transforms/augmentation/spatial/random_affine.py:451]:
```suggestion
            ' for the scaling factor. Refer to the documentation for more information.'
```

**Comment by fepegar** [comment]
@romainVala is this clearer?

**Comment by AminAlam** [comment]
> Thanks, @AminAlam. I've suggested some changes.

Thanks for the corrections. I committed the changes.

**Comment by fepegar** [comment]
Thanks, for your contribution, @AminAlam!

@all-contributors please add @AminAlam for doc

**Pull request merged**